bridge-to-the-moon/src/offchain/deploy.js

```diff
@@ -0,0 +1,96 @@
+const fs = require('fs')
```

Comment:
why are we writing our own deploy script with compilation, etc?

Reply:
That is for offchain deployment stuff. It is being deployed to the parity chain, there might be an easier way to do it but thats how it got it to work so far. The truffle stuff deploys stuff to onchain